### PR TITLE
Dev amd outlier normalization

### PR DESCRIPTION
**Pull request by youngyoonii**
- 기존에 있던 실험용 config file 삭제
- AMD 시리즈에 Outlier Normalization 추가
- 관련 기본 config 구조 수정

### CODE REVIEW

File: mdistiller/distillers/AMD_norm.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+from ._base import Distiller
+from ._common import get_feat_shapes, SimpleAdapter, make_zscore_mask, make_gaussian_std_mask, normalize_outlier_artifacts
+
```
Comment (nda111) [inline mdistiller/distillers/AMD_norm.py:13]: `import`, `class` 사이에는 개행 두번 하세요.
```python
import ...
# 한 줄
# 두 줄
class ...
```

File: mdistiller/distillers/AMD_norm.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+from ._base import Distiller
+from ._common import get_feat_shapes, SimpleAdapter, make_zscore_mask, make_gaussian_std_mask, normalize_outlier_artifacts
```
Comment (nda111) [inline mdistiller/distillers/AMD_norm.py]: `import` 너무 길어지면 괄호로 묶어서 정리하세요.
```python
from ._common import (
    get_feat_shapes,
    SimpleAdapter,
    make_zscore_mask,
    make_gaussian_std_mask,
    normalize_outlier_artifacts,
)
```

File: mdistiller/distillers/AMD_recon.py [hunk 3]
```diff
@@ -3,7 +3,7 @@
 import torch.nn.functional as F
 
 from ._base import Distiller
-from ._common import get_feat_shapes, SimpleAdapter, ReconMHA
+from ._common import get_feat_shapes, SimpleAdapter, ReconMHA, make_zscore_mask, make_gaussian_std_mask, normalize_outlier_artifacts
```
Comment (nda111) [inline mdistiller/distillers/AMD_recon.py]: 여기서 `import` 정리하세요.

File: mdistiller/distillers/AMD_recon.py [hunk 4]
```diff
@@ -69,9 +70,22 @@ def forward_train(self, image, target, **kwargs):
...
+                case _:
+                    raise NotImplementedError(self.af_type)
+            
             match self.af_recon_type:
                 case 'recon_mha':
-                    (recon_f_t, _) , outlier_mask, _ = self.reconstructor_dict[f"recon_mha_{m_l:03d}"](f_t)
+                    if self.af_norm_outlier:
+                        f_t = normalize_outlier_artifacts(f_t, outlier_mask)
+                    ignore_outliers = not self.af_norm_outlier
+                    (recon_f_t, _) , outlier_mask, _ = self.reconstructor_dict[f"recon_mha_{m_l:03d}"](f_t, outlier_mask, ignore_outliers=ignore_outliers)
```
Comment (nda111) [inline mdistiller/distillers/AMD_recon.py]: line 너무 길어지니까 두 줄로 나누면 어떨까요?
```python
reconstructor = self.reconstructor_dict[f"recon_mha_{m_l:03d}"]
(recon_f_t, _) , outlier_mask, _ = reconstructor(f_t, outlier_mask, ignore_outliers=ignore_outliers)
```

File: mdistiller/engine/cfg.py [hunk 5]
```diff
@@ -85,8 +85,9 @@ def dump_cfg(cfg, show: bool=False):
 CFG.AMD.INPUT_SIZE = [224, 224]
 CFG.AMD.AF = CN()
 CFG.AMD.AF.ENABLE = True
+CFG.AMD.AF.OUT_NORM = False
```
Comment (nda111) [inline mdistiller/engine/cfg.py]: 왜 이름이 `OUT_NORM`인가요?

**Review by nda111 — CHANGES_REQUESTED** [review]
몇 가지 수정사항, 제안사항 있습니다. 확인해주세요 @youngyoonii

File: mdistiller/distillers/AMD_norm.py (hunk 1 above)
Comment (youngyoonii) [reply mdistiller/distillers/AMD_norm.py:7]: from ._common import get_feat_shapes, SimpleAdapter, make_zscore_mask, make_gaussian_std_mask, normalize_outlier_artifacts
이렇게 주루룩 쓰는게 취향이신가요
아니면 엔터 쳐서 정리하는게 취향이신가요

File: mdistiller/engine/cfg.py (hunk 5 above)
Comment (youngyoonii) [reply mdistiller/engine/cfg.py]: `ARTIFACT_NORM`으로 수정했습니다.

**Comment by nda111** [comment]
수정 감사합니다. 머지해주세요. @youngyoonii
